packages/swap/src/handlers/__tests__/__snapshots__/openSwapDepositChannel.test.ts.snap

```diff
@@ -334,8 +589,104 @@ exports[`openSwapDepositChannel creates channel with ccmParams and stores it in
 }
 `;
 
-exports[`openSwapDepositChannel creates channel with fill or kill params and stores it in the database 1`] = `
+exports[`openSwapDepositChannel > creates channel with fill or kill params and stores it in the database 1`] = `
 [
+  [
+    {
```

Comment:
i think these are all the calls to the mock noiw - maybe we need to reset mocks in between tests to restore the old behaviour?

Reply:
good catch